front/packages/ui/src/player/video.tsx

```diff
@@ -109,7 +109,7 @@ const Video = forwardRef<VideoRef, VideoProps>(function Video(
 				}
 				// when video file is invalid, audio is undefined
 				selectedAudioTrack={{ type: SelectedTrackType.INDEX, value: audio?.index ?? 0 }}
-				textTracks={subtitles?.map((x) => ({
+				textTracks={subtitles?.filter(x => !!x.link).map((x) => ({
```

Comment:
I can't build, publish, or test this myself. However it follows identical logic to the web player, which I did test, so I think it'll work here.

Reply:
yeah sounds good, exoplayer (current player on android) doesn't support pgs & co anyways